benchmarks/2022/benchmarks-schema.yaml

```diff
@@ -16,6 +16,12 @@ definitions:
       isToppedOut:
         description: isToppedOut is a boolean value that describes whether a benchmark's latter deciles repeat a value of 100 or, in the case of inverse measures, a value of 0.
         type: boolean
+      isToppedOutByProgram:
+        description: isToppedOutByProgram is a boolean value that describes whether a benchmark's latter deciles repeat a value of 100 or, in the case of inverse measures, a value of 0.
+        type: boolean
+      isHighPriority:
+        description: whether the measure is a high priority measure or not
```

Comment:
here too

Reply:
High priority measures include the following categories of measures: Outcome, Appropriate Use, Patient Experience, Patient Safety, Efficiency measures, Care coordination. These measures are determined as the highest priority by CMS and at least one High Priority measure must be submitted out of the 6 required quality measures in QPP MIPS.